stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java

```diff
@@ -35,6 +55,111 @@ public void bindingWithViewModel() {
     }
 
+    /**
+     * @see <a href="https://getstream.io/chat/docs/node/channels_view_new/">Channels View</a>
+     */
+    public class Channels extends Fragment {
+
+        ChannelsView channelsView;
+
+        public void bindingWithViewModel() {
+            // Get ViewModel
+            FilterObject filter = Filters.and(
+                    Filters.eq("type", "messaging"),
+                    Filters.in("members", singletonList(ChatDomain.instance().getCurrentUser().getId()))
+            );
+            int limit = 30;
+
+            ChannelsViewModelFactory factory = new ChannelsViewModelFactory(filter, ChannelsViewModel.DEFAULT_SORT, limit);
+            ChannelsViewModel viewModel = new ViewModelProvider(this, factory)
+                    .get(ChannelsViewModel.class);
+
+            // Bind it with ChannelsView
+            ChannelsViewModelBinding.bind(viewModel, channelsView, getViewLifecycleOwner());
+        }
+
+        public void handlingChannelActions() {
+            channelsView.setChannelInfoClickListener((channel) -> {
+                // Handle Channel Info Click
+            });
+
+            channelsView.setUserClickListener((user) -> {
+                // Handle Member Click
+            });
+        }
+
+        public void handlingUserInteractions() {
+            channelsView.setChannelItemClickListener((channel) -> {
+                // Handle Channel Click
+            });
+
+            channelsView.setChannelLongClickListener((channel) -> {
+                // Handle Channel Long Click
+                return true;
+            });
+        }
+
+        public void customizingDefaultViews() {
+            // Create loading view and layout params
+            ProgressBar loadingView = new ProgressBar(getContext());
+            FrameLayout.LayoutParams layoutParams = new FrameLayout.LayoutParams(
+                    FrameLayout.LayoutParams.WRAP_CONTENT,
+                    FrameLayout.LayoutParams.WRAP_CONTENT,
+                    Gravity.CENTER
+            );
+            channelsView.setEmptyStateView(loadingView, layoutParams);
+
+            // Create empty state view and use default layout params
+            TextView emptyStateView = new TextView(getContext());
+            emptyStateView.setText("No channels available");
+            channelsView.setEmptyStateView(emptyStateView);
+
+
+            // Set custom item separator drawable
+            channelsView.setItemSeparator(R.drawable.stream_ui_divider);
+
+            // Add separator to the last item
+            channelsView.setShouldDrawItemSeparatorOnLastItem(true);
+        }
+
+        public void customViewHolderFactory() {
+            class CustomChannelListItemViewHolderFactory extends ChannelListItemViewHolderFactory {
+                @Override
+                public int getItemViewType(@NotNull ChannelListItem item) {
+                    // Override together with createViewHolder() to introduce different view holder types
```

Comment:
Should we include such snippets, if there is not any customization? Maybe use an out-of-box factory and leave customization use case for another doc or cookbook?

Reply:
IMO providing the code for the whole customization is too much for CMS docs, so I decided just to show methods available to override, but I agree that this might be a part of cookbook. @zsmb13 what do you think?

Reply:
I like the snippet here like this for now, as it provides an idea of what you can do, and then a full customization sample can go somewhere else.